instance/models/openedx_instance.py

```diff
@@ -274,9 +274,14 @@ def get_load_balancer_configuration(self):
         """
         if not self.active_appserver:
             return self.get_preliminary_page_config(self.ref.pk)
+        ip_address = self.active_appserver.server.public_ip
+        if not ip_address:
+            # The active appserver doesn't have a public IP address.  This means that the server
+            # has been terminated, so we don't show the preliminary page in this case, and simply
+            # deconfigure the backend instead.
+            return [], []
```

Comment:
What would the user see in the UI in that case? Is there be a way to signal this to the user? It would feel wrong to just silently deconfigure the backen, without giving any clue to the user that something was off.